src/components/BookDetail/BookDetail.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from '@nextui-org/react'
+import PriceTable from '@/components/BookDetail/PriceTable'
+import FavoriteChip from '@/components/FavoriteChip'
+import BookImage from '@/components/BookImage'
+import { type BookInfo } from '@/hooks/queries/useBookInfo'
+
+export default function BookDetail({ isbn, bookInfo }: { isbn: string; bookInfo: BookInfo }) {
+  return (
+    <div className="flex flex-row h-full w-full m-10 gap-6">
+      <div className="flex flex-col gap-4">
+        <BookImage image={bookInfo.image} title={bookInfo.title} />
+        <div className="flex flex-row gap-1 items-center">
+          <FavoriteChip customClasses="text-base" markedAsFavoriteCount={0} />
+          <Button radius="full" color="primary" variant="light">
+            + Add to favorites
```

Comment:
[SUG] Probably we should use an icon instead of the `+` character

Reply:
Doesn't that add more overhead? I mean because the icon is brought from a library, so it means more things to load, and visually there shouldn't be much difference in this case 🤔

Reply:
That's right, but something you have to consider is that lucide icons doesn't add wrappers around it like vuetify or other libraries, these icons are straight svg inserted in the dom. Also, since our pages are very simple (they are not full of components rendering every time) it shouldn't be much of a difference in performance.

Either way, is up to you, for me personally I prefer using icons but for now using the ascii character is fine too 🦘

Reply:
ok, you convinced me jaja